tests/test_dynamodb_checkpointer.py

```
@@ -0,0 +1,340 @@
+"""Tests for DynamoDB checkpointer."""
+
+import asyncio
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)